eq-publisher/src/eq_schema/Block.js

```diff
@@ -19,7 +19,7 @@ const getLastPage = flow(
 const processPipedText = ctx =>
   flow(
     convertPipes(ctx),
-    getInnerHTMLWithPiping
+    unescapePiping
```

Comment:
I guess we should add some tests to cover this.

I think we want to get the inner html for title. So is that affected with this change?